azurelinuxagent/common/protocol/extensions_goal_state_from_vm_settings.py

```diff
@@ -165,7 +161,7 @@ def _parse_simple_attributes(self, vm_settings):
         #         "correlationId": "9a47a2a2-e740-4bfc-b11b-4f2f7cfe7d2e",
         #         "inSvdSeqNo": 1,
         #         "extensionsLastModifiedTickCount": 637726657706205217,
-        #         "extensionGoalStatesSource": "Fabric",
+        #         "extensionGoalStatesSource": "FastTrack",
```

Comment:
Updated the vmSettings test data and sample code to FastTrack (since now we ignore Fabric)